### ISSUE TITLE
Nested modal instances

### ISSUE THREAD

**Issue by jayarnielsen** [pineconellc/angular-foundation#248]
I have a rare case where I need to open another modal from within a modal instance. It works, but the translucent background doesn't cover up the previous modal, so you can still click on it, thereby opening even more modal instances. It would be great if the modal stack would cover up any existing modals.

**Comment by timothyswt** [comment]
I would really suggest folling a ui-router stateful modal paradigm (opening the modal on the onEnter handler)

https://github.com/timothyswt/ng-gulp-hapi/blob/master/client/views/main/modal/modal.js

then you can transition to and from modals without having them overlay on top of eachother (bad practice)

**Comment by jayarnielsen** [comment]
Yeah, I'm already doing that elsewhere, but it's not appropriate in this instance. I ended up going a completely different route. I recognize it's not the best design, but there are cases where it would be useful.
